sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/BlobChangeFeedExtensions.cs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+namespace Azure.Storage.Blobs.ChangeFeed
+{
+    /// <summary>
+    /// BlobChangeFeedExtensions.
+    /// </summary>
+    public static class BlobChangeFeedExtensions
+    {
+        /// <summary>
+        /// GetChangeFeedClient.
+        /// </summary>
+        /// <param name="serviceClient"></param>
+        /// <returns><see cref="BlobChangeFeedClient"/>.</returns>
+        public static BlobChangeFeedClient GetChangeFeedClient(this BlobServiceClient serviceClient)
+        {
+            return new BlobChangeFeedClient(serviceClient);
+        }
+
+        /// <summary>
+        /// Builds a DateTimeOffset from a segment path.
+        /// </summary>
+        internal static DateTimeOffset? ToDateTimeOffset(this string segmentPath)
+        {
+            if (segmentPath == null)
+            {
+                return default;
+            }
+            segmentPath = segmentPath.Trim('/');
+            string[] splitPath = segmentPath.Split('/');
+            return new DateTimeOffset(
+                year: int.Parse(splitPath[2], CultureInfo.InvariantCulture),
```

Comment:
I think we should guard against index out of range.

Reply:
Fixed.